jobs-builder/README.md

```diff
@@ -0,0 +1,36 @@
+# Overview
+
+Manage the Jenkins jobs with help of the [Jenkins Job Builder](https://docs.openstack.org/infra/jenkins-job-builder).
+
+# Getting started
+
+First of all, you need to install Jenkins Job Builder in your environment. The
+instructions can be found [here](https://docs.openstack.org/infra/jenkins-job-builder/installation.html).
+
+To use the Jenkins Job Builder it is needed a configuration file which contains
+the Jenkins URL, user and token API to manage Jenkins, among other information
+that sets the tool's behavior. That file can copied from the `jjb.conf.template`
+template then filling out the fields marked with *XXX*.
+
+Bootstrapping your environment:
+```bash
+pip install --user jenkins-job-builder
+cp jjb.conf.template jjb.conf
```

Comment:
Does `pip` install the plugin files to the current directory?

Reply:
Nope, I provide the `jjb.conf.template` file as part of this PR